RadixWallet/Features/AssetsFeature/Components/HelperViews/AssetResourceDetails.swift

```diff
@@ -13,6 +13,7 @@ struct AssetResourceDetailsSection: View {
 		let validatorAddress: ValidatorAddress?
 		let resourceName: Loadable<String?>?
 		let currentSupply: Loadable<String?>
+		let divisibility: Loadable<UInt8?>?
```

Comment:
Using optional Loadables seems a bit much, no? IF the inner optional is nil, that means that e.g. divisibility is not present, but what does it mean if the entire property is nil? That divisibility is not relevant for this resource? If so, that makes some sense, but it makes less sense for resourceName.

Reply:
yes, for example NFTs would have the `divisibility` set to nil. However, if the `Lodable` is present, we still have to make its inner value optional since it may not be present for a given fungible resource.

Reply:
>, if the Lodable is present, we still have to make its inner value optional since it may not be present for a given fungible resource.

@matiasbzurovski if the value is not present what do you show in the resource details view? 0? or just empty?

Reply:
empty